lib/WP_Auth0_Lock.php

```diff
@@ -0,0 +1,213 @@
+<?php
```

Comment:
Nothing actually new in this file:

- Renamed
- Removed deprecated methods
- Added `render()` method from `renderAuth0Form()` in `templates/login-form.php`